src/hooks/useWalletSectionsData.ts

```diff
@@ -27,6 +29,13 @@ export default function useWalletSectionsData({
   } = useLegacyNFTs({
     address: accountAddress,
   });
+
+  const walletsWithBalancesAndNames = useWalletsWithBalancesAndNames();
```

Comment:
i was looking at the useWallets hook and it seems like it may be very heavy due to its use of useInitializeWallet. This has me wondering if we should limit our usage of useWallets, and instead get the necessary data directly from the redux selector when possible.

the reason why this line of code got me wondering about it is that useWalletsWithBalancesAndNames calls useWallets, as well as another instance on line 21 of this file, so that computation is duplicated.